src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -620,18 +620,18 @@ private static Assembly ResolveAssembly(object sender, ResolveEventArgs args)
         }
 
         /// <summary>
-        /// Gets a <see cref="DurableOrchestrationClient"/> using configuration from a <see cref="OrchestrationClientAttribute"/> instance.
+        /// Gets a <see cref="DurableClient"/> using configuration from a <see cref="DurableClientAttribute"/> instance.
         /// </summary>
         /// <param name="attribute">The attribute containing the client configuration parameters.</param>
-        /// <returns>Returns a <see cref="DurableOrchestrationClient"/> instance. The returned instance may be a cached instance.</returns>
-        protected internal virtual IDurableOrchestrationClient GetClient(OrchestrationClientAttribute attribute)
+        /// <returns>Returns a <see cref="DurableClient"/> instance. The returned instance may be a cached instance.</returns>

```

Comment:
NIT: the documentation references the concrete type, but the type signature references the interface. I would make the documentation match the signature..